iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/execution/operator/process/TableIntoOperator.java

```diff
@@ -111,10 +111,15 @@ protected TsBlock tryToReturnResultTsBlock() {
 
   @Override
   public long ramBytesUsed() {
```

Comment:
I think we still need to update the `calculateMaxPeekMemory`, `calculateMaxReturnSize` and `calculateRetainedSizeAfterCallingNext`